src/containers/AppBody.js

```diff
@@ -6,10 +6,13 @@ import { setState } from '../actions/app';
 import SnippetArea from './SnippetArea';
 import AnnotationDisplay from './AnnotationDisplay';
 
+const API_URL = process.env.API_URL;
+
 class AppBody extends Component {
   componentDidMount() {
     const { dispatch, snippetKey } = this.props;
-    axios.get("../snippet.json")
+    const [user, snippet] = snippetKey.split("/");
+    axios.get(`${API_URL}/users/${user}/snippets/${snippet}`)
       .then((res) => {
         dispatch(setState(res.data));
       });
```

Comment:
I'm thinking that kinda needs to be it's own story, because I think the entire view should change when the snippet is broken.